accounts/tests/views/test_user.py

```diff
@@ -0,0 +1,43 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from accounts.models import User
+from unittest.mock import patch
+
+
+class UserTestCase(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create(email="user@staff.com")
+
+    @patch("accounts.serializers.UserCreateSerializer.create")
+    def test_post_user(self, mock_method):
+        url = reverse("users-list")
+        self.client.force_authenticate(self.user)
+        response = self.client.post(
+            url,
+            {
+                "email": "test@user.com",
+            },
+            format="json",
+        )
+        self.assertTrue(mock_method.called)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_method_not_allowed(self):
+        url = reverse("users-list")
```

Comment:
put, patch, and delete are applied to `users-detail` ([docs](https://www.django-rest-framework.org/api-guide/routers/#defaultrouter)), so these would continue to pass even if we added them! this used to be inconsistent in our tests but fixed in #57 and #58